planner/plan.go

```diff
@@ -27,11 +27,18 @@ type Plan struct {
 	// StartStage defines the actions that should be taken when
 	// starting (running) the application.
 	// Ex: python main.py
-	StartStage *Stage `json:"start_stage,omitempty"`
+	StartStage *EntrypointStage `json:"start_stage,omitempty"`
 }
 
 type Stage struct {
 	Command string `cue:"string" json:"command"`
+	Image   string `cue:"string" json:"image"`
+}
+
+type EntrypointStage struct {
```

Comment:
`StartStage`? Entrypoint may not be immediately intuitive imo.

Also, can this inherit the `Stage` struct?